perf/benchmark/configs/istio/security_authn/config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
can we rename the folder to `security_peer_authn` to be consistent with other tests?